improver_tests/nbhood/circular_kernel/test_CircularNeighbourhood.py

```diff
@@ -353,37 +305,35 @@ def test_single_point_on_corner(self):
         )
         for index, slice_ in enumerate(expected_centroid):
             expected[index][0:3] = slice_
-        ranges = 3
-        result = CircularNeighbourhood(weighted_mode=True).apply_circular_kernel(
-            self.cube, ranges
-        )
+        radius = 6000
+        result = Neighbourhood("circular", weighted_mode=True).run(self.cube, radius)
         self.assertArrayAlmostEqual(result.data, expected)
 
 
 class Test_run(IrisTest):
 
-    """Test the run method on the CircularNeighbourhood class."""
+    """Test the run method on the Neighbourhood class."""
 
-    RADIUS = 6100
+    RADIUS = 4000
```

Comment:
The tests below had to be changed slightly as the radius before was actually larger than the domain.